util/imagetools/loader.go

```diff
@@ -292,6 +295,23 @@ func (l *loader) scanSBOM(ctx context.Context, fetcher remotes.Fetcher, r *resul
 					if err != nil {
 						return nil, err
 					}
+
+					if layer.MediaType == "application/vnd.in-toto.spdx+dsse" {
+						var dsse struct {
+							Payload string `json:"payload"`
+						}
+						if err := json.Unmarshal(dt, &dsse); err != nil {
+							return nil, err
+						}
+
+						decoded, err := base64.StdEncoding.DecodeString(dsse.Payload)
```

Comment:
Should the payload be trusted if signatures (format) have not been verified?

Reply:
is there a mechanism in place to throw warnings for `imagetools inspect` commands? if so, adding a warning that the statements haven't been verified would help here

Reply:
could potentially use `logrus` here to print a warning along the lines of `warning: sbom statements are not verified` until if/when this code checks signatures. I haven't seen many examples in the code base other than https://github.com/docker/buildx/blob/a6228ed78f3d591e56014d82c37185de4939e4c5/util/cobrautil/cobrautil.go#L37